External/HIP/utils/compare_image.py

```diff
@@ -0,0 +1,69 @@
+#!/bin/python3
+import argparse
+import os
+import sys
+
+try:
+    import cv2
+    import numpy as np
+    from skimage.metrics import structural_similarity as ssim
+except ImportError:
+    print("One or more required packages are not installed. Please install them using the command:")
```

Comment:
Should that be checked during cmake configuration phase instead?

Reply:
You can have both. An enabled but failing test will just create unnecessary noise if tests are set up w/o blender. If blender is a requirement for the test suite, it's cmake's job to check for its presence, and enable/disable affected tests, IMO.

Up to you.